parse_emails/parse_emails.py

```diff
@@ -38,8 +38,8 @@ def __init__(self, file_path, max_depth=3, parse_only_headers=False, file_info='
 
     def get_file_type(self, file_type):
 
-        mime = magic.Magic()
-        if not file_type:
+        mime = magic.Magic(keep_going=True)
+        if not file_type or file_type == 'AMUSIC Adlib Tracker':
```

Comment:
Can you explain the keep_going=True?
Also, who gives EmailParser the wrong file_type?